scripts/deploy.coffee

```diff
@@ -0,0 +1,25 @@
+# Description:
+#   Causes heart attacks.
+#
+# Commands:
+#   hubot deploy to production - Deploy Ralph to production.
+
+apps = [
+  'argos',
+  'levelup-order-ahead-production',
+  'levelup-production',
+  'order-ahead-network-production',
+  'plutus-production'
+]
+
+emojis = [
+  ':+1:',
```

Comment:
Consider `:party_poop:`, `:partydumpster:`, and `:party_try_not_to_cry:` as valuable emotes